gnmi/gnmi.go

```diff
@@ -75,6 +75,12 @@ type Server struct {
 
 // New creates and registers a reference gNMI server on the given gRPC server.
 //
+// To avoid having to specify the Target field in gNMI requests in order to comply
```

Comment:
would it be possible to change the implementation so this isn't needed?
before to calling cache Subscribe populate the target?

Reply:
Yes that might be possible and would increase the usability of the fake gNMI server. I'll refactor the code from the interceptor instead of this change in a different PR.